.github/workflows/release.yml

```diff
@@ -100,12 +101,138 @@ jobs:
         with:
           repository-url: https://test.pypi.org/legacy/
 
+  # Job that builds conda package for distribution
+  build-conda-package:
+    name: Build Conda Package
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set Up Miniconda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          channels: conda-forge,defaults
+          auto-update-conda: true
+          python-version: "3.11"
+          activate-environment: "test"
+
+      - name: Store conda-bld Location
+        run: echo "CONDA_BLD_PATH=/usr/share/miniconda/envs/test/conda-bld" >> $GITHUB_ENV
+
+      - name: Build Conda Package
+        shell: bash -l {0}
+        run: |
+          python --version
+          conda install -y conda-build conda-verify anaconda-client
+          conda build .
+          conda build --output .
+
+      - name: Test Built Package from Local Channel
+        shell: bash -l {0}
+        run: |
+          echo $CONDA_BLD_PATH
+          python --version
+          conda create -n test-env -y -c file://$CONDA_BLD_PATH space_packet_parser
+          conda activate test-env
+          python -c "import space_packet_parser"
+          which spp
+          cat $(which spp)
+          spp --version
+          conda deactivate
+
+      - name: Upload Conda Build Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: conda-package
+          path: ${{ env.CONDA_BLD_PATH }}/**/space_packet_parser-*.conda
+
+  # Job that uploads conda package to Anaconda.org on the LASP channel
```

Comment:
These two jobs for the official / test channels are nearly identical. I'd suggest merging them together except for the difference. A suggestion would be to add the if statement to the final step and just have two different steps one for each different upload path.

Reply:
Good call. I was patterning this from the PyPI upload jobs, which require running in different environments for OIDC with PyPI. Anaconda doesn't have that restriction though.